src/sonic-host-services/scripts/caclmgrd

```diff
@@ -375,6 +379,8 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
         # Walk the ACL tables
         for (table_name, table_data) in self._tables_db_info.items():
+            if not table_data:
+                continue
```

Comment:
Maybe we should also log a warning message here stating that there was no data for table X.

Reply:
done

Reply:
It would be helpful to log the name of the table here.

Reply:
done